docs/pbc/all/request-for-quote/use-cases-request-for-quote-processes.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Use cases: Request for Quote processes"
+description: This document describes the use cases for Requests for Quote.
+last_updated: Nov 15, 2022
+template: concept-topic-template
+---
+
+
+## Use case 1: Long waiting time until Request for Quote (RFQ) gets processed
+
+The [request for quote](/docs/pbc/all/request-for-quote/request-for-quote.html) process is currently not entirely efficient because it is manual. As a result, sometimes, customers have to wait for a long time until their RFQ gets approved or denied. 
+
+### Solution
+
+To solve this issue, merchants must react faster to quote requests. For example, they can take the following measures:
+* Clarify the essential details upfront with customers.
+* Automate the RFQ process where possible: create assignment criteria and define workflows with predefined thresholds per customer if needed.
+* Notify customers about the processing of their RFQs through, for example, email with provided estimated delivery dates.
+* Notify merchants about new or updated RFQ through, for example, email.
+
+## Use case 2: Marketplace customers must create separate carts and RFQs for each merchant
+
+In Marketplace, if customers have several items from different merchants in their cart, they have to send an RFQ for each item separately. This means they have to create separate carts for products from different merchants they want to request quotes from. 
+
+### Solution
+
+To solve this issue, Spryker needs to introduce Marketplace support for RFQ. To achieve this, the following actions are required:
```

Comment:
```suggestion
To solve this issue, Spryker needs to introduce Marketplace support for RFQ. To achieve this, the following actions are required:
```
Unsure of meaning, does this mean that the feature has not yet been introduced or that this can be done already?

Reply:
To my best knowledge, Marketplace does not support RFQ

Reply:
Got it. I'm just unsure why we're advertising a feature for marketplace that's not there yet - to clarify this is just me being curious, wondering abt the reasoning no more!